src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -118,10 +122,11 @@ public static void merge(List<RepoConfiguration> repoConfigs, List<RepoConfigura
     }
 
     /**
-     * Sets all {@code RepoConfiguration} in {@code configs} to have {@code formats} set.
+     * Sets {@code formats} to {@code RepoConfiguration} in {@code configs} if {@code formats} is empty.
```

Comment:
`if {@code formats} is empty.` this part is wrong. Shouldn't use `{@code ...}` for internal implementation.

Perhaps `if its formats is empty.` will do.